src/common/assert.cc

```diff
@@ -159,9 +172,19 @@ namespace ceph {
       if (!g_assert_context->_conf->fatal_signal_handlers) {
 	g_assert_context->_log->dump_recent();
       }
-    }
 
-    abort();
+      // bypass the abort?
+      const auto supressions = get_str_list(
+	g_assert_context->_conf.get_val<std::string>("ceph_assert_supresssions"));
+      should_abort = std::none_of(
+	std::begin(supressions), std::end(supressions),
+	[file, line](const auto& supression) {
+	  return supression == fmt::format("{}:{}", file, line);
+        });
+    }
+    if (should_abort) {
+      abort();
+    }
   }
```

Comment:
Completely irrelevant in context of this PR,
but shouldn't we take common parts of __ceph_assert_fail and __ceph_assertf_fail together?

Reply:
Yeah, I agree. They are asking for a unification.